### ISSUE TITLE
[Grida Canvas] Gradient editor `delete` a11y

### ISSUE THREAD

**Issue by softmarshmallow** [gridaco/grida#409]
Gradient editor is missing `delete` key hot key binding.
to do this we'll need to handle key events from the main editor instance, or re-design our gradient editor to be provider pattern, as the key handling should be universal (not bounded to certain component)

<img width="954" height="883" alt="Image" src="https://github.com/user-attachments/assets/b4a743d3-195b-4a6b-bba5-45848e37be17" />
